lumigator/python/mzai/jobs/inference/model_clients.py

```diff
@@ -150,7 +150,16 @@ def _chat_completion(
 
 class HuggingFaceModelClient(BaseModelClient):
     def __init__(self, config: InferenceJobConfig):
-        self._pipeline = pipeline(**config.hf_pipeline.model_dump())
+        pipeline_kwargs = config.hf_pipeline.model_dump()
+        if hasattr(config.hf_pipeline, "tokenizer") and config.hf_pipeline.tokenizer is not None:
```

Comment:
Maybe this is simpler, if no other falsy objects could be stored here:

```suggestion
        if hasattr(config.hf_pipeline, "tokenizer") and config.hf_pipeline.tokenizer:
```